src/io/socket.rs

```diff
@@ -1,6 +1,7 @@
+use crate::runtime::driver::op::Op;
```

Comment:
Maybe socket.rs could go somewhere else? It doesn't exactly represent IO like the other files in this directory do. (I think?)

Reply:
I'd agree, I just need to figure out what to do with it